docs/source/markdown/podman-container.unit.5.md.in

```diff
@@ -0,0 +1,502 @@
+
+If set to `ALL`, Podman will unmask all the paths that are masked or made read-only by default.
+
+The default masked paths are /proc/acpi, /proc/kcore, /proc/keys, /proc/latency_stats, /proc/sched_debug, /proc/scsi, /proc/timer_list, /proc/timer_stats, /sys/firmware, and /sys/fs/selinux.
+
+The default paths that are read-only are /proc/asound, /proc/bus, /proc/fs, /proc/irq, /proc/sys, /proc/sysrq-trigger, /sys/fs/cgroup.
+
+@@option quadlet:user
+
+@@option quadlet:userns.container
+
+@@option quadlet:volume
+
+@@option quadlet:workdir
+
+
+# SERVICE TYPE
+
+By default, the generator sets `Type=notify` for `.container` units.
+
+This can be overridden by explicitly setting `Type=` in the `[Service]` section.
+
+For one-shot containers (e.g., init tasks), use:
+
+```
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+```
+
+Refer to `systemd.service(5)` for full details on service types and related behaviors.
+
+# INSTALL SECTION
+
+To ensure a container starts on boot, include an `[Install]` section:
+
+```
+[Install]
+WantedBy=multi-user.target
+```
+
+Only `Alias=`, `WantedBy=`, `RequiredBy=`, and `UpheldBy=` keys are supported.
+
+# TIMEOUTS
+
+Container startup may exceed systemd’s default 90s timeout (e.g., when pulling images). Use:
+
+```
+[Service]
+TimeoutStartSec=900
+```
+
+Note: `TimeoutStartSec` is ignored for `Type=oneshot`.
+
+# NETWORK DEPENDENCIES
+
+Quadlet adds:
+
+- `After=network-online.target` (for root units)
+- `After=podman-user-wait-network-online.service` (for user units)
+
+To disable this, add:
+
+```
+[Quadlet]
+DefaultDependencies=false
+```
+
+# RESOURCE NAMING
+
+By default, the container is named `systemd-<unitname>`. Use `ContainerName=` to override.
+
+Avoid using systemd specifiers like `%N` in resource names—they break inter-resource linking.
+
+# TEMPLATE UNITS
+
+Quadlet supports templated container units, e.g., `foo@.container` creates `foo@.service`.
+
+You can instantiate with:
+
+```bash
+systemctl start foo@bar.service
+```
+
+You may also symlink instances:
+
+```bash
+ln -s foo@.container foo@bar.container
+```
+
+Use drop-ins for instance-specific customization:
+
+```
+foo@bar.container.d/10-override.conf
+```
+
+# DROP-IN FILES
+
+Quadlet supports drop-in configuration in `.container.d/` directories.
+
+For example, a drop-in:
+
+```
+test.container.d/10-extra.conf
+```
+
+can override or extend the main unit file.
+
+Drop-ins follow the same override and merging behavior as systemd units.
+
+# EXAMPLE
+
+Minimal container unit:
+
+```
+[Unit]
+Description=A minimal container
+
+[Container]
+Image=quay.io/centos/centos:latest
+Exec=sleep 60
+
+[Install]
+WantedBy=multi-user.target
+```
+
+# SEE ALSO
+
+[systemd.unit(5)](https://www.freedesktop.org/software/systemd/man/systemd.unit.html),
+[systemd.service(5)](https://www.freedesktop.org/software/systemd/man/systemd.service.html),
+[systemd-analyze(1)](https://www.freedesktop.org/software/systemd/man/latest/systemd-analyze.html),
```

Comment:
why is systemd-analyze linked here? the main page described how to debug with it not this file